### PR TITLE
Feature/sign up with google

### PR DESCRIPTION
**Pull request by KseniaKoms**
add signup with google logic
photo_2022-11-17_03-36-44
photo_2022-11-17_03-36-34

### CODE REVIEW

File: apps/freelance/src/app/app.tsx [hunk 1]
```diff
@@ -1,21 +1,29 @@
 import { StyledApp } from './app.styled';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import LoginPage from '@pages/login-page/login-page';
 import SignupPage from '@pages/signup-page/signup-page';
+import store from '../redux/store';
+import { GoogleOAuthProvider } from '@react-oauth/google';
 import RecoverPassFirstPage from '@pages/recoverpass-first-page/recoverpass-first-page';
```
Comment (lyudad) [inline apps/freelance/src/app/app.tsx:8]: Снимок экрана 2022-11-17 в 19 31 03

File: libs/components/src/lib/form-container/form-container.tsx [hunk 2]
```diff
@@ -60,7 +62,7 @@ export function FormContainer({
         </Link>
       </StyledParagraph>
       {isSignForm && (
-        <StyledButton id="googleBtn" buttonSize="md" buttonColor="blue">
+        <StyledButton onClick={() => handleSuccess()} id="googleBtn" buttonSize="md" buttonColor="blue">
```
Comment (lyudad) [inline libs/components/src/lib/form-container/form-container.tsx]: onClick={handleSuccess}

Comment (KseniaKoms) [reply libs/components/src/lib/form-container/form-container.tsx]: fixed

File: apps/freelance/src/app/app.tsx (hunk 1 above)
Comment (KseniaKoms) [reply apps/freelance/src/app/app.tsx:8]: fixed

**Pull request merged**